examples/managers/mid_agreement_payments.py

```diff
@@ -61,19 +61,22 @@ async def idle_work(context: WorkContext) -> str:
 
 async def main():
     logging.config.dictConfig(DEFAULT_LOGGING)
-    payload = RepositoryVmPayload("9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae")
+    debit_notes_interval = timedelta(minutes=3)
+    payment_timeout = timedelta(minutes=30)
+    payloads = [
+        RepositoryVmPayload("9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"),
```

Comment:
one thing -> I know it's mostly irrelevant but I'd note what the image here is...

Reply:
We do not have support for tags in core, I changed the image to alpine and added an comment